docs/source/tutorial/commercial_paper.md

```diff
@@ -961,7 +861,7 @@ DigiBank's `buy.js` application is very similar in structure to MagnetoCorp's
   * **Transaction**: the invoked transaction is `buy` rather than `issue`
 
     ```JavaScript
-    `const buyResponse = await contract.submitTransaction('buy', 'MagnetoCorp', '00001'...);`
+    const buyResponse = await contract.submitTransaction('buy', 'MagnetoCorp', '00001'...);
     ```
 
```

Comment:
const wallet = new FileSystemWallet('../identity/user/balaji/wallet'); should be
const wallet = await Wallets.newFileSystemWallet('../identity/user/balaji/wallet');

const buyResponse = await contract.submitTransaction('buy', 'MagnetoCorp', '00001'...); should be
const buyResponse = await contract.submitTransaction('buy', 'MagnetoCorp', '00001', 'MagnetoCorp', 'DigiBank', '4900000', '2020-05-31');

And we probably need to explain what those last 2 parameters do '4900000', '2020-05-31.

Reply:
done.